java/compaction-status-store/src/main/java/sleeper/compaction/status/DynamoDBUtils.java

```diff
@@ -51,6 +55,14 @@ public static void initialiseTable(
         try {
             CreateTableResult result = dynamoDB.createTable(request);
             LOGGER.info("Created table {}", result.getTableDescription().getTableName());
+            dynamoDB.updateTimeToLive(new UpdateTimeToLiveRequest()
```

Comment:
This adds the time-to-live functionality to Dynamo tables created using this class, but this is only used for tests. I think ttl needs adding to the cdk code that creates the Dynamo job status table.